backend/src/services/integration-auth/integration-sync-secret.ts

```diff
@@ -207,6 +207,12 @@ const syncSecretsGCPSecretManager = async ({
         }
       );
 
+      if (!secrets[key].value) {
+        logger.warn(
+          `[GCP_SECRET_SYNC_1]: Empty secret value for key: ${key} and integration appId: ${integration.appId}`
```

Comment:
Follow this format for logs:

```
function-name: text text text [key=value] [key2=value]
```

This will help you query for logs much easier 

Example:

```
syncSecretsGCPSecretManager: send secret to gcp where [key=${key}] and integration [appId= ${integration.appId}]
```